.env.example

```diff
@@ -1,5 +1,6 @@
-NEXT_APP_PROJECT_ID="Appwrit Cloud project ID"
-NEXT_APP_DATABASE_ID="Appwrit Cloud database ID"
-NEXT_APP_USER_COLLECTION="Appwrit Cloud user collection ID"
-NEXT_APP_POSTS_COLLECTION="Appwrit Cloud post collection ID"
-NEXT_APP_BASE_URL="base url of the web app"
\ No newline at end of file
+NEXT_PUBLIC_PROJECT_ID="651d73a6e5781b730a33"
```

Comment:
Please solve the merge conflicts for this file. Also, I don't want to expose the dev keys to GitHub so please don't add that.

Reply:
No problem, I just removed the env variables